.github/workflows/pull-request.yml

```diff
@@ -63,6 +63,17 @@ jobs:
         with:
           xcode-version: '15.0.1'
 
+      - name: "Install rust toolchain (Linux)"
+        if: matrix.os-type == 'linux'
+        run: sudo apt install rustc build-essential -y
+
+      - name: "Install rust toolchain (Macos)"
+        if: matrix.os-type == 'macos'
+        run: brew install rustup
+          
+      - name: "Install wasm-pack"
+        run: cargo install wasm-pack
+          
```

Comment:
We need to add the same lines to the release and docs `yml` files as well.

Reply:
done